### DISCUSSION TITLE
Game crashes on borderless

### DISCUSSION THREAD

**Discussion by expiteRz** [ppy/osu#19660] [Q&A] [answered]
### Type

Crash to desktop

### Bug description

The game crashes when I changed the window mode from Fullscreen to Bolderless. Tried to restart the game but it will be closed immediately.

### Screenshots or videos

The video is a moment after changing the window mode.

https://user-images.githubusercontent.com/51064411/183583600-e6a55d8a-c16f-4b0b-bf2a-1d3c36c9e7bd.mp4

### Version

2022.731.1

### Logs

[database.log](https://github.com/ppy/osu/files/9287765/database.log)
[network.log](https://github.com/ppy/osu/files/9287766/network.log)
[performance.log](https://github.com/ppy/osu/files/9287767/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9287768/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9287769/updater.log)

**Comment by peppy** [comment, marked as answer]
Going to be nvidia shadowplay or something else on your system. The crash is a native (driver level) one.

**Reply by expiteRz** [reply]
I've enabled Instant Replay for the other game. I confirmed the game is launching without it. Thanks for the answer so fast, and sorry for upsetting you.

**Reply by peppy** [reply]
No need for apology, there was no upset here.

I'm not sure if we'll get to figuring out a solution for this before nvidia do (it's quite likely something at their end), but we are looking to switch the graphics backend to use Direct X / vulkan by default on windows within the coming months, which should fix this issue and many others.
